### PR TITLE
Moving Automatic sec update to pots

### PR DESCRIPTION
**Pull request by rigoucr**

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -304,6 +311,43 @@ jobs:
...
+      - run:
+          name: setup-environment-vars
+          command: |
+            if [ -f "./.circleci/scripts/set-environment" ]; then
+              ./.circleci/scripts/set-environment
+            else
+              ./vendor/fourkitchens/pots/scripts/set-environment
+            fi
+      - add_ssh_keys
+      - run: terminus -n auth:login --machine-token="$TERMINUS_TOKEN"
```
Comment (generalredneck) [inline config.yml:340]: Why do you need terminus?

File: config.yml [hunk 2]
```diff
@@ -304,6 +311,43 @@ jobs:
       - deploy-to-pantheon-env:
           environment: 'live'
 
+
+  security_updates_check:
+    <<: *defaults
+    docker:
+      # quay.io/pantheon-public/build-tools-ci:8.x-php${PHPVERSION}
+      - image: quay.io/pantheon-public/build-tools-ci:8.x-php<< pipeline.parameters.php-version >>
```
Comment (generalredneck) [inline config.yml:319]: If you don't need terminus, using  straight defaults here would be best. See the static tests job for an example.

File: config.yml [hunk 3]
```diff
@@ -323,9 +367,27 @@ jobs:
 
 workflows:
   version: 2
+
+  scheduled_update_check:
+    when: << pipeline.parameters.run-sec-updates-check >>
+    jobs:
+      - static_tests
+      - build
+      - security_updates_check:
```
Comment (generalredneck) [inline config.yml:376]: static tests and build really don't need to be here. They will be run when the Pull Request is made.

**Review by generalredneck — CHANGES_REQUESTED** [review]
I think this is too heavy. I'm pretty sure that this job can be solo and on lighter infrastructure making it more performant.

**Comment by rigoucr** [comment]
Great feedback, thnk you!! I think I have applied all your suggestions,

File: config.yml (hunk 3 above)
Comment (rigoucr) [reply config.yml:377]: Thank you ! Removed those reqs

File: config.yml (hunk 2 above)
Comment (rigoucr) [reply config.yml:319]: I'm going to keep the image because We are going to change it anyway in the near future when we implement the Config export

**Review by rigoucr — COMMENTED** [review]
Thx

File: config.yml (hunk 1 above)
Comment (rigoucr) [reply config.yml]: Removed

**Review by generalredneck — APPROVED** [review]
We will need to think on how this is going to work for other hosts in the future. So far, it sounds like the plan is only for Pantheon. We will need to make it work for ACLI and possibly drush in the future.

**Comment by rigoucr** [comment]
@generalredneck I'm going to restore the branch meanwhile we create a official release , because scr is currently using this branch to pull the package.

**Pull request merged**